packages/hydroxide/build.sh

```diff
@@ -12,6 +13,7 @@ termux_step_make() {
 	export GOPATH=$TERMUX_PKG_BUILDDIR
 
 	cd $TERMUX_PKG_SRCDIR
+	patch -u imap/mailbox.go -i mailbox.go.patch
```

Comment:
This is not needed, or rather returns with error.

Also the current `mailbox.go.patch` is broken because it does not show what file it should be applied to.